### ISSUE TITLE
Update of OpenBSD7

### ISSUE THREAD

**Issue by schurzi** [lavabit/robox#305]
The current version of OpenBSD is 7.6 and 7.4 has beeen deprecated two weeks ago. This means packages are no longer easily installable and the box is practivcally broken now.

An update to 7.5 seems to be prepared, maybe it would be better to switch direktly to 7.6. Is there already a planned update or are you accepting contributions for an update?

**Comment by ThomasWaldmann** [comment]
Yup, can confirm. No borgbackup testing on openbsd7 currently due to this.

**Comment by ThomasWaldmann** [comment]
This is how the box can be fixed for now:

https://github.com/borgbackup/borg/pull/8554/files

**Comment by TheTumultuousUnicornOfDarkness** [comment]
> This is how the box can be fixed for now:
> 
> https://github.com/borgbackup/borg/pull/8554/files

I see, https://ftp.eu.openbsd.org/pub/OpenBSD contains packages for all OpenBSD version.
Good idea, thanks!
